onnxruntime/contrib_ops/cuda/bert/rotary_embedding_impl.cu

```diff
@@ -105,10 +111,10 @@ Status LaunchRotaryEmbeddingKernel(
   ORT_ENFORCE(head_size <= max_threads_per_block,
               "Rotary embedding dim must be <= max_threads_per_block");
 
-  int tpb = (head_size + 31)/32*32;
+  int tpb = (head_size + 31) / 32 * 32;
 
   const dim3 block(tpb);
-  const dim3 grid(num_heads, sequence_length, batch_size);
+  const dim3 grid(num_heads, 1024, batch_size * (sequence_length / 1024 + 1));
```

Comment:
batch_size * (sequence_length / 1024 + 1) looks odd to me. Shall it be (batch_size * sequence_length + 1023) / 1024